### PR TITLE
VIX-3293 Fixture Effect has some linq queries that prevent it from working at the group level

### PR DESCRIPTION
**Pull request by johncbaur**
VIX-3293 Fixture Effect has some linq queries that prevent it from working at the group level

### CODE REVIEW

File: Modules/Effect/Fixture/Fixture/FixtureModule.cs [hunk 1]
```diff
@@ -408,7 +408,7 @@ private void RenderCurve(Curve curve, FixtureFunction function, CancellationToke
 		private FixtureFunction GetFunction(string functionName)
 		{
 			// Return the first fixture function that matches the name
-			return _fixtureFunctions.Single(fn => fn.Name == functionName);
+			return _fixtureFunctions.First(fn => fn.Name == functionName);
 		}
```
Comment (jeffu231) [inline Modules/Effect/Fixture/Fixture/FixtureModule.cs:412]: Are there any cases where First would not find a value and could throw an exception? The other changes used first or default and have checks so more of a sanity question here.

Comment (johncbaur) [reply Modules/Effect/Fixture/Fixture/FixtureModule.cs:412]: I don't think so...but I have been spectacularly wrong before.
I can add some defensive code.

Comment (jeffu231) [reply Modules/Effect/Fixture/Fixture/FixtureModule.cs:412]: No need to add defensive logic in unless it is warranted.  I will go ahead and merge.

**Comment by johncbaur** [comment]
I just merged some updates to this ticket.

**Pull request merged**